imx8qm/default.nix

```diff
@@ -1,13 +1,22 @@
 # SPDX-FileCopyrightText: 2022 Unikie
 
 { config ? import ../../spectrum/nix/eval-config.nix {} }: config.pkgs.callPackage (
-{ stdenvNoCC, util-linux, jq, mtools }:
+{ stdenvNoCC, util-linux, jq, mtools, enableKvms ? false }:
+
+with config.pkgs;
 
 let
   inherit (config);
   uboot = config.pkgs.ubootIMX8QM;
-  spectrum = import ../../spectrum/release/live { inherit (config); };
+  spectrum = import ../../spectrum/img/live { inherit (config); };
```

Comment:
This indeed depends on which upstream branch you follow.
Current main has img/live and in-progress wayland has this refactored to release/live
As these build configs are downstream from spectrum, we need to adapt.
Further discussions? Please take to spectrum mailing list for comments.

Reply:
Yes, futher discussions: understand that current state of development requires pointing to "release" folder, but end or intermediate goal is to get rid from tiiuae forks at all and use upstreams of spectrum and nixpkgs-spectrum, do you confirm that?

Reply:
Yes, in general, target is to avoid diverging forks of spectrum and nix-pkgs repositories. Or keep the amount of delta patches to minimum.

Regarding this repository - spectrum-config-imx8 - there’s no upstream. This manages device specific imx8 build configuration out of spectrum tree. If you have questions on specifics, read this thread https://spectrum-os.org/lists/archives/spectrum-devel/87wna3n5l5.fsf@alyssa.is/T/#t and continue discussion there.